src/components/Pagination/index.tsx

```diff
@@ -43,7 +43,33 @@ export const getPaginationInfo = ({
   };
 };
 
-export const PaginationContext = React.createContext(null);
+export type PaginationContextValue<PageType = number> = {
```

Comment:
Does this mean that `PageType` must be type number or inherited ? Or is this just a default value for this generics ?

If it is just a default value for this generics, I fail to see the use for a generic here as I can't see where you would use something else than a number. I must be missing something here.

Reply:
Its the default for the generic, and I was thinking about a `string` type provided by the API so we had to type the `PageType` with a string. But that may be overkill at the moment yeah.

Reply:
If the defaults are great (which they are) then there are no problems this being overkill I think !